src/ninja.cc

```diff
@@ -218,8 +218,8 @@ class NinjaGenerator {
  private:
   void PopulateNinjaNodes(const std::vector<NamedDepNode>& nodes) {
     ScopedTimeReporter tr("ninja gen (eval)");
-    for (auto const& node : nodes) {
-      PopulateNinjaNode(*node.second);
+    for (auto const& [symbol, depnode] : nodes) {
```

Comment:
Can we assume C++17? (I don't know off the bat what compiler is used to build production Kati)

Reply:
Kati uses C++17. https://github.com/google/kati/blob/master/Makefile.ckati#L84

Reply:
Well, *that*, I certainly didn't expect!